### PR TITLE
Add an "all-in-one" image

### PR DESCRIPTION
**Pull request by mab27**
"All-in-one" docker image for st2. Under images/stackstorm-all

### CODE REVIEW

**Comment by mab27** [comment]
@warrenvw this is the PR for the all-in-one image.

File: images/stackstorm-all/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+## How to use this pack ?:
+
+- You first need to git clone this repo.
+```
+sudo git clone https://github.com/mab27/docker_st2.git
+```
+
+### Build:
+ - First change the ```ST2_PASSWORD``` variable in the Dockerfile. (You may skip this and stay with default password if you using this image for testing purposes only, for production usage though, you may need to change it).
+ - Use the docker ```build``` command to create the docker image and give it a name (in this example we are naming the image stackstorm/st2_aio)
```
Comment (warrenvw) [inline images/stackstorm-all/README.md]: Perhaps use `stackstorm/stackstorm-all` instead of `stackstorm/st2_aio`?

File: images/stackstorm-all/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ 	- The simplest is to go to the directory that contains the Dockerfile and type in the following command (replace ```.``` by the path to Dockerfile if you are not in that directory): 
+```
+sudo docker build -t stackstorm/st2_aio .
+```
+ - This will start building the image, it may take a whilety the first time as it will pull the ubuntu image (approximatly 10 mn). Once it is done, you can check by typing:
```
Comment (warrenvw) [inline images/stackstorm-all/README.md]: whilety -> while

File: images/stackstorm-all/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,56 @@
+####################################
+# Build the image:
+#
+#   docker build -t stackstorm/st2_aio .
+#	You need to be in the dirctory that contains the Dockerfile.
```
Comment (shusugmt) [inline images/stackstorm-all/Dockerfile]: I think these general comments that are not directly related to the content of Dockerfile can be moved and merged into `README.md`
